### ISSUE TITLE
improvements for "Album info"

### ISSUE THREAD

**Issue by da-anda** [phil65/skin.estuary#145]
I think it would be good to show a list of local songs in the album info screen, even if there is no metadata for that album.

Second improvement would be a simple "play" button to be able to play or enqueue the album from this screen.

**Comment by DaveTBlake** [comment]
+1 play button

There is album information missing: moods, styles, themes, and (record) label. There are info labels for all these that could usefuly be displayed.

Regards showing local songs, especially if the album has not been scraped and there is no track listing, I would like to do core changes so that the track info is always populated with local songs combined with scrapped tracks listing, and there is a clear link to the song (if there is one) and the ability to add to playlist.  I know that the skin could show songs here already as a separate list, but a combined control seems nicer.

**Comment by da-anda** [comment]
I always wondered why anybody would care about a scraped song list instead of simply showing the local ones. What's the point in showing a scraped track list? Who cares about this?

**Comment by DaveTBlake** [comment]
Yes I think a scraped track list is pretty useless too, especially if it is just a list and even if you have the song you can't do anything with it (e.g. play or see more info about it).

I'll open a thread in music corner and see if I can find anyone that uses it.

**Comment by da-anda** [comment]
@DaveTBlake any feedback about the scraped track list?

**Comment by DaveTBlake** [comment]
@da-anda generally users agree scraped track list  is useless, but as always the number of replies is small. I guess a change for Leia, where the underlying data can changed too.

Meanwhile I would have loved to see the missing information go in v17: genres, moods, styles, themes, (record) label, rating (a scraped value not same as user rating) and votes (for that rating), playcount and last played date.

**I consider that an Estuary bug** but don't have the skinning skills to make fixing it efficient.

**Comment by phil65** [comment]
you seem to have a weird definition of "bug"..
Infodialogs will get reworked for v18. For v17 these changes would be too intrusive.

**Comment by DaveTBlake** [comment]
>you seem to have a weird definition of "bug"..
Software not doing what it is required to do.
Information visible in the default skin in v16, not shown in the default skin in v17. Yeap that's a bug.
